src/Search.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import SearchButton from "./SearchButton";
 
 const Search = () => {
```

Comment:
In the Bookings.js component, you are providing a search callback to this component. But you are not receiving it in here. Did you forget?